src/main/java/tatracker/model/student/UniqueStudentList.java

```diff
@@ -28,6 +29,24 @@
     private final ObservableList<Student> internalList = FXCollections.observableArrayList();
     private final ObservableList<Student> internalUnmodifiableList =
             FXCollections.unmodifiableObservableList(internalList);
+    private Comparator<Student> alphabetically = new Comparator<Student>() {
+        @Override
+        public int compare(Student student, Student other) {
+            return (student.getName().toString()).compareTo(other.getName().toString());
+        }
+    };
+    private Comparator<Student> ratingAscending = new Comparator<Student>() {
+        @Override
+        public int compare(Student student, Student other) {
+            return (student.getRating().toString()).compareTo(other.getRating().toString());
+        }
+    };
+    private Comparator<Student> ratingDescending = new Comparator<Student>() {
+        @Override
+        public int compare(Student student, Student other) {
+            return (-1) * (student.getRating().toString()).compareTo(other.getRating().toString());
+        }
+    };
```

Comment:
This is cool, but I think you could actually do ratingAscending.reversed() haha!

Reply:
In fact prob no need to have this! In your sort function, just use ratingAscending.reversed()!

Reply:
haha that's good to keep in mind for the future. I think for now can keep this la